grails-app/services/org/pih/warehouse/inventory/StockMovementService.groovy

```diff
@@ -1168,7 +1168,7 @@ class StockMovementService {
         Location location = requisitionItem?.requisition?.origin
 
         // Qty Available
-        List<AvailableItem> availableItems = productAvailabilityService.getAvailableBinLocations(location, requisitionItem.product)
+        List<AvailableItem> availableItems = productAvailabilityService.getAllAvailableBinLocations(location, [requisitionItem.product])
```

Comment:
Use the single product method instead of method for list of products

Reply:
@awalkowiak Done.